packages/ui/src/form/DynamicFormField/DynamicInput.tsx

```diff
@@ -1,10 +1,12 @@
 import { FormFieldItem } from '@bigcommerce/checkout-sdk';
+import classNames from 'classnames';
 import { isDate, noop } from 'lodash';
 import React, { FunctionComponent, memo, useCallback } from 'react';
 import ReactDatePicker from 'react-datepicker';
 
-import { useLocale } from '@bigcommerce/checkout/locale';
+// import { useLocale } from '@bigcommerce/checkout/locale';
```

Comment:
Why is this line commented?

Reply:
Please see the comment below.

Reply:
Could you please update this line?

Reply:
@bc-peng what do you mean? Is `useLocale` working now?

Reply:
Yes, we have finalised the `locale` package.
The implementation is at https://github.com/bigcommerce/checkout-js/pull/1245.